script/testing/util/test_server.py

```diff
@@ -160,22 +172,32 @@ def restart_db(self):
         self.stop_db()
         self.run_db()
 
+    def print_output(self, filename):
```

Comment:
I don't think we need this method. I think I created one outside the class cause this doesn't use anything specific to the class.